haystack/schema.py

```diff
@@ -1502,17 +1502,25 @@ def load(cls, load_dir: Union[str, Path], **read_csv_kwargs):
             "gold_answers",
             "gold_documents_id_match",
             "gold_offsets_in_documents",
+            "gold_offsets_in_contexts",
             "gold_answers_exact_match",
             "gold_answers_f1",
             "gold_answers_sas",
             "gold_answers_match",
             "gold_contexts_similarity",
             "offsets_in_document",
+            "offsets_in_context",
             "document_ids",
             "custom_document_ids",
             "gold_document_contents",
         ]
-        converters = dict.fromkeys(cols_to_convert, ast.literal_eval)
+
+        def safe_literal_eval(x: str) -> Any:
+            if x == "":
```

Comment:
Can x be None? Would that be a problem?

Reply:
It would be a problem, but it can't be. Empty entries in CSV (e.g. `,,`) are translated into an empty string.

Reply:
See `test_load_evaluation_result_w_empty_document_ids`. It writes out a `None` value for `document_ids`, it doesn't break.